core-plugin/testSrc/com/google/gct/idea/debugger/actions/CloudDebugHelpActionTest.java

```diff
@@ -0,0 +1,54 @@
+package com.google.gct.idea.debugger.actions;
+
+import static org.junit.Assert.assertEquals;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+import com.google.gct.idea.debugger.actions.CloudDebugHelpAction;
+
+import com.intellij.CommonBundle;
+import com.intellij.openapi.actionSystem.AnAction;
+import com.intellij.openapi.actionSystem.AnActionEvent;
+import com.intellij.openapi.actionSystem.Presentation;
+import com.intellij.openapi.util.IconLoader;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.MockitoAnnotations;
+
+import java.lang.Override;
+
+public class CloudDebugHelpActionTest {
```

Comment:
@RunWith(MockitoJUnitRunner.class)

T^hen you don't need the setUp method below